.travis.yml

```diff
@@ -125,4 +125,5 @@ cache:
     - validator/nodejs/node_modules
     - validator/webui/node_modules
     - validator/java/bazel-installer
+    - $HOME/.m2
```

Comment:
what does this mean?

Reply:
It caches the maven artifacts that are downloaded during install.